docs/en/setup/backend/aws-firehose-receiver.md

```diff
@@ -32,5 +32,7 @@ The following blogs demonstrate complete setup process for AWS S3 and API Gatewa
 ## Notice
 
 1. Only OpenTelemetry format is supported (refer to [Metric streams output formats](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-metric-streams-formats.html))
-2. A proxy(e.g. Nginx, Envoy) is required in front of OAP's Firehose receiver to accept HTTPS requests from AWS Firehose through port `443` (refer to [Amazon Kinesis Data Firehose Delivery Stream HTTP Endpoint Delivery Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html).
+2. According to HTTPS requirement by AWS Firehose(refer to [Amazon Kinesis Data Firehose Delivery Stream HTTP Endpoint Delivery Specifications](https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html), users have two options
+  - A proxy(e.g. Nginx, Envoy) is required in front of OAP's Firehose receiver to accept HTTPS requests from AWS Firehose through port `443`. (Recommended based on the general security policy)
+  - Set `aws-firehose/enableTLS=true` and `acceptProxyRequest=true` at OAP side to accept requests from firehose directly.
```

Comment:
I think we can enable `acceptProxyRequest=true` always and remove this config to minimize the configurations, users don't need to set this extra config. It has no side effect right?

Also for the tls setting, we should also mention the cert/key files settings here.

Reply:
Updated.

Reply:
> I think we can enable acceptProxyRequest=true always and remove this config to minimize the configurations, users don't need to set this extra config. It has no side effect right?

About this, I think `acceptProxyRequest=true` is harmless from our use case. @pg-yang Could you recheck about this?

Reply:
Consider a case, users use a gateway to accept TCP over TLS(participate in TLS handshake), and then transport TCP without TSL to a firehose receiver, the configuration will be `enableTLS =false,acceptProxyRequest=true`. I know the case is uncommon.

Reply:
This seems never recommended by us, I think.
This configuration is not straight forward, we knew this only we read source codes of America. I don't think users would know.

Reply:
acceptProxyRequest will have same value with enableTLS: 

https://github.com/apache/skywalking/pull/10684/commits/07f3de50f18edae8e3c789839a73f4722c435974#diff-8f7b4b497c144f8ccbc29c6ccfa29d06281ee8eeab86ccc03979e6223bba9f6eR78